client/web/src/user/settings/codeHosts/CodeHostItem.tsx

```diff
@@ -49,6 +50,7 @@ export const CodeHostItem: React.FunctionComponent<CodeHostItemProps> = ({
     isUpdateModalOpen,
     toggleUpdateModal,
     onDidUpsert,
+    useGitHubApp,
```

Comment:
We could provide a default value to leverage that the `useGitHubApp` might be optional.
```suggestion
    useGitHubApp = false,
```